src/components/ItemCard.css

```diff
@@ -12,6 +12,13 @@
 }
 
 .frame {
-
   border: var(--color-icons) solid;
 }
+
+img {
+  height: 5rem;
+}
```

Comment:
Let's be a bit more specific here. Styling the `img` element directly would affect every images in the project.
Even though we divide the css in smaller files, the css is not scoped to this component, is loaded and affects globally.

Hence, let's give this `img` a class (e.g. Item-card__image) or something like this instead

Reply:
I was too eager to see how it looks in LiveServer and missed to apply some proper class. Thx!